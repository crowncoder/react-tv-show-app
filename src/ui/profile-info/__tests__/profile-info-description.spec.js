import React from 'react';
import Enzyme, { shallow } from 'enzyme'
import EnzymeAdapter from 'enzyme-adapter-react-16'
import ProfileInfoDescription from '../profile-info-description'

Enzyme.configure({ adapter: new EnzymeAdapter() });

describe('<ProfileInfoDescription /> unit test', () => {
    it('renders correctly', () => {
        const description = 'A simplate description';
        const wrapper = shallow(<ProfileInfoDescription description={description} />);
        expect(wrapper.find('div').prop('dangerouslySetInnerHTML'))
            .toEqual({ __html: 'A simplate description' });
        expect(wrapper).toMatchSnapshot();
    });
});