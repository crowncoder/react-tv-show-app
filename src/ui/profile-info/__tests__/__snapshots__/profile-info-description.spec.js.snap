// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileInfoDescription /> unit test renders correctly 1`] = `
<div
  className="profileInfo__des"
  dangerouslySetInnerHTML={
    Object {
      "__html": "A simplate description",
    }
  }
/>
`;
